hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -135,7 +135,7 @@ def test_get_calibration_from(self):
 class Test_quantification:
 
     def setUp(self):
-        s = EDSTEMSpectrum(np.ones([2, 1024]))
+        s = EDSTEMSpectrum(np.ones([2, 2, 1024]))
         energy_axis = s.axes_manager.signal_axes[0]
```

Comment:
I’m not sure because we have beam_energy and beam_current so I was trying to keep consistent with the other names used in the metadata.

> On 11 Apr 2016, at 15:49, Duncan Johnstone <notifications@github.com> wrote:
> 
> In hyperspy/tests/signal/test_eds_tem.py <https://github.com/hyperspy/hyperspy/pull/913#discussion_r59206385>:
> 
> > @@ -135,7 +135,7 @@ def test_get_calibration_from(self):
> >  class Test_quantification:
> >  
> >      def setUp(self):
> > -        s = EDSTEMSpectrum(np.ones([2, 1024]))
> > +        s = EDSTEMSpectrum(np.ones([2, 2, 1024]))
> >          energy_axis = s.axes_manager.signal_axes[0]
> Ok - only question is minor, but do we thing "beam_area" or "probe_area"?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/hyperspy/hyperspy/pull/913/files/ff223181b7d44327f1bbf263d33e8bd9580a5748#r59206385>